typedoc.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Ion JS Library",
```

Comment:
Nit: should this just be `ion-js` or `Ion JS`?

Reply:
Done.